src/kirby/components/list/list.event.ts

```diff
@@ -0,0 +1,8 @@
+export interface LoadOnDemandEvent {
+  complete: (disableLoadOnDemand?: boolean) => void;
+}
+
+export interface LoadOnDemandEventData {
```

Comment:
```suggestion
export interface LoadOnDemandEvent {
```

Reply:
There is a other interface in this file called `LoadOnDemandEvent`. 

This `LoadOnDemandEventData` is ment as a wrapper for {N}'s `LoadOnDemandListViewEventData` interface.

Reply:
There is a other interface in this file called `LoadOnDemandEvent`. 

This `LoadOnDemandEventData` is ment as a wrapper for {N}'s `LoadOnDemandListViewEventData` interface.